### PR TITLE
LPS-139257 content dashboard assert

### PR DESCRIPTION
**Pull request by lucassantanasantos**
LPS-139257 content dashboard assert
https://issues.liferay.com/browse/LPS-139257

### CODE REVIEW

**Comment by lucassantanasantos** [comment]
@CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformanceWithoutAC.testcase [hunk 1]
```diff
@@ -91,11 +98,53 @@ definition {
...
-		// TODO LPS-139257 MetricsInContentDashboardACDisabled pending implementation
+		task ("Check in Content Dashboard and open Content Performance Panel") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+
+			Click.clickNoMouseOver(locator1 = "ContentDashboard#ROW_PANEL_WC");
+
+			Click(locator1 = "ContentPerformance#ICON_VIEW_METRICS");
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentperformance/ContentPerformanceWithoutAC.testcase:136]: This should be included in a macro, try to not have clicks within the testcase file.

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
Also the SF is failing, please review it, thanks 😊

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
@lucassantanasantos The SF failure seems to be related to your PR, could you please review?

**Pull request closed without merging**